upgrade_test.py

```diff
@@ -206,7 +209,10 @@ def _upgrade_node(self, node, upgrade_sstables=True, new_scylla_repo=None, new_v
             scylla_yaml_updates.update({"consistent_cluster_management": True})
 
         if self.params.get("enable_tablets_on_upgrade"):
-            scylla_yaml_updates.update({"experimental_features": ["tablets", "consistent-topology-changes"]})
+            scylla_yaml_updates.update({"enable_tablets": True})
+
+        if self.params.get("enable_force_`gossip_topology_changes"):
```

Comment:
typo here in param - so it looks this path was never tested

Reply:
fixed